projects/solid/package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "@maskito/solid",
+    "version": "0.0.1",
+    "description": "",
+    "license": "MIT",
```

Comment:
Change license to `Apache` (similar to all other Maskito's packages):

https://github.com/taiga-family/maskito/blob/52824cd49487e2cab9c46038294777c015c65efa/package.json#L21